site/src/pages/DeploySettingsPage/NotificationsPage/NotificationsPage.tsx

```diff
@@ -0,0 +1,269 @@
+import { Loader } from "components/Loader/Loader";
+import { Stack } from "components/Stack/Stack";
+import { TabLink, Tabs, TabsList } from "components/Tabs/Tabs";
+import {
+  castNotificationMethod,
+  methodIcons,
+  methodLabels,
+  type NotificationMethod,
+} from "modules/notifications/utils";
+import { Section } from "pages/UserSettingsPage/Section";
+import { deploymentGroupHasParent } from "utils/deployOptions";
+import { pageTitle } from "utils/page";
+import { useDeploySettings } from "../DeploySettingsLayout";
+import OptionsTable from "../OptionsTable";
+
+type MethodToggleGroupProps = {
+  templateId: string;
+  options: NotificationMethod[];
+  value: NotificationMethod;
+};
+
+const MethodToggleGroup: FC<MethodToggleGroupProps> = ({
+  value,
+  options,
+  templateId,
+}) => {
+  const queryClient = useQueryClient();
+  const updateMethodMutation = useMutation(
+    updateNotificationTemplateMethod(templateId, queryClient),
+  );
+
+  return (
+    <ToggleButtonGroup
+      exclusive
+      value={value}
+      size="small"
+      aria-label="Notification method"
+      css={styles.toggleGroup}
+      onChange={async (_, method) => {
+        await updateMethodMutation.mutateAsync({
+          method,
+        });
+        displaySuccess("Notification method updated");
+      }}
+    >
+      {options.map((method) => {
+        const Icon = methodIcons[method];
+        const label = methodLabels[method];
+        return (
+          <Tooltip key={method} title={label}>
+            <ToggleButton
+              value={method}
+              css={styles.toggleButton}
+              onClick={(e) => {
+                // Retain the value if the user clicks the same button, ensuring
+                // at least one value remains selected.
+                if (method === value) {
+                  e.preventDefault();
+                  e.stopPropagation();
+                  return;
+                }
+              }}
+            >
+              <Icon aria-label={label} />
+            </ToggleButton>
+          </Tooltip>
+        );
+      })}
+    </ToggleButtonGroup>
+  );
+};
+
+export const NotificationsPage: FC = () => {
+  const [searchParams] = useSearchParams();
+  const { deploymentValues } = useDeploySettings();
+  const [templatesByGroup, dispatchMethods] = useQueries({
+    queries: [
+      {
+        ...systemNotificationTemplates(),
+        select: selectTemplatesByGroup,
+      },
+      notificationDispatchMethods(),
+    ],
+  });
+  const ready =
+    templatesByGroup.data && dispatchMethods.data && deploymentValues;
+  const tab = searchParams.get("tab") || "events";
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("Notifications Settings")}</title>
+      </Helmet>
+      <Section
+        title="Notifications"
+        description={
+          <>
+            Control delivery methods for notifications on this deployment.
+            Notifications may be disabled in your{" "}
```

Comment:
I think we need to clarify here that these are not disabled at a deployment level.
In fact, I'd just remove this line entirely. WDYT @stirby?

Reply:
We could alternatively use:

> "You may opt-out of specific notifications in your [profile settings](coder.com)."

This sounds more individual than the former.

Reply:
The first line is describing what this page is for, we need to keep it in.
The second line you suggested could be slightly modified to

> "You may opt-out of receiving specific notifications in your [profile settings](https://github.com/coder/coder/pull/coder.com)."

reads better to me; I feel it's still oddly placed here, though, and IMHO we should drop it.

Reply:
I figure administrators are going to see this page and wonder where to disable notifications. I like your latest rendition best but we can drop it if you think it's too out of place.

Reply:
Lets remove it and if we see users get confused, we can put more thoughts on this copy.